client/admin/apps/AppDetailsPage.js

```diff
@@ -0,0 +1,136 @@
+import React, { useState, useCallback } from 'react';
+import { Button, ButtonGroup, Icon, Avatar, Box, Divider, Chip, Margins, Skeleton } from '@rocket.chat/fuselage';
+
+import Page from '../../components/basic/Page';
+import { useRoute } from '../../contexts/RouterContext';
+import PriceDisplay from './PriceDisplay';
+import { AppStatus } from './AppStatus';
+import { useTranslation } from '../../contexts/TranslationContext';
+import { useAppInfo } from './hooks/useAppInfo';
+import { AppMenu } from './AppMenu';
+import { useLoggedInCloud } from './hooks/useLoggedInCloud';
+
+const objectFit = { objectFit: 'contain' };
+
+function AppDetailsPageContent({ data, setModal, isLoggedIn }) {
+	const t = useTranslation();
+
+	const {
+		iconFileData = '',
+		name,
+		author: { name: authorName, homepage, support } = {},
+		description,
+		categories = [],
+		version,
+		price,
+		purchaseType,
+		pricingPlans,
+		iconFileContent,
+		installed,
+		bundledIn,
+	} = data;
+
+
+	return <><Box display='flex' flexDirection='row' mbe='x20' w='full'>
+		<Avatar style={objectFit} size='x120' mie='x20' url={iconFileContent || `data:image/png;base64,${ iconFileData }`}/>
+		<Box display='flex' flexDirection='column' justifyContent='space-between' flexGrow={1}>
+			<Box fontScale='h1' fontWeight='500'>{name}</Box>
+			<Box display='flex' flexDirection='row' color='hint' alignItems='center'>
+				<Box fontScale='p2' mie='x4'>{`${ t('By') } ${ authorName }`}</Box>
+			|
+				<Box mis= 'x4'>{`${ t('Version') } ${ version }`}</Box>
+			</Box>
+			<Box display='flex' flexDirection='row' alignItems='center' justifyContent='space-between'>
+				<Box flexGrow={1} display='flex' flexDirection='row' alignItems='center'>
+					<AppStatus mie='x4' app={data} setModal={setModal} isLoggedIn={isLoggedIn}/>
+					{!installed && <PriceDisplay mis='x4' purchaseType={purchaseType} pricingPlans={pricingPlans} price={price} showType={false}/>}
+				</Box>
+				{installed && <AppMenu app={data} setModal={setModal} isLoggedIn={isLoggedIn} />}
+			</Box>
+		</Box>
+	</Box>
+	<Divider />
+	<Box display='flex' flexDirection='column'>
+
+		<Margins block='x12'>
+			<Box fontScale='h1' textTransform='uppercase'>{t('Categories')}</Box>
```

Comment:
We need to check if those uppercases are needed now. Also, is it not strange too much `h1`-like headings in a single page?